docker-compose-full.yml

```diff
@@ -45,7 +45,7 @@ services:
       - s3
       - setup
     environment:
-      - LANGUAGE_SERVER_HOST=localhost:3023
+      - LANGUAGE_SERVER_HOST=ws://localhost:3023
```

Comment:
Happy for this to get merged, but I'd like this instead to go into Exercism Config gem rather than into this file as a next step. So please add an issue for this when it gets merged.